app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id, dependent: :destroy
+  has_many :comments, foreign_key: :author_id, dependent: :destroy
+  has_many :likes, foreign_key: :author_id, dependent: :destroy
```

Comment:
- [ ] As the previous reviewer suggested the likes and comments associations should use `user_id`, instead of `author_id`. Alternatively, since the foreign key is same name as the model, we can omit the `foreign_key` definition.

  ```
  has_many :comments, dependent: :destroy
  has_many :likes, dependent: :destroy
  ```